NineChronicles.Headless.Tests/GraphTypes/StandaloneMutationTest.cs

```diff
@@ -607,12 +608,19 @@ public async Task CombinationEquipment(Address avatarAddress, int recipeId, int
 
         [Theory]
         [MemberData(nameof(ItemEnhancementMember))]
-        public async Task ItemEnhancement(Address avatarAddress, Guid itemId, Guid materialId, int slotIndex)
+        public async Task ItemEnhancement(Address avatarAddress, Guid itemId, List<Guid> materialIds, int slotIndex)
         {
             var playerPrivateKey = new PrivateKey();
+            var materialQuery = new StringBuilder("[");
+            foreach (var materialId in materialIds)
+            {
+                materialQuery.Append($" \"{materialId}\"");
```

Comment:
Is it okay when `materialIds` has multiple ids? (there is no comma, ',')

Reply:
I've checked manually query without comma works fine. :smiley: